### DISCUSSION TITLE
Script.SQL to .MDF/.LOG

### DISCUSSION THREAD

**Discussion by pingucas** [community/community#135089] [Programming Help]
### Body

Good afternoon

I'm downgrading a sql 2019 database with 8GB to sql 2014, I've already created the script (it ended up with 32GB), now I'm trying to open it in Studio Management to run the script and create the MDF and Log and gives the error in the print, can anyone help 
I'm doing this on a machine with a lot of resources 32GB RAM Ryzen7 etc.
![error](https://github.com/user-attachments/assets/d54eecff-9998-42ea-b005-67a25cc790cc)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by geniuszly** [comment]
By itself, the error message `System.OutOfMemoryException` in SQL Server Management Studio (SSMS) occurs because the script you are trying to load is extremely large (32GB). Even though your machine has significant resources, SSMS might not be able to handle such a large file in its current environment. 
To somehow eliminate or fix this problem, you can try to divide the script into smaller parts and run them alternately. Or instead of loading the script into SSMS, you can use the sqlcmd utility to execute the script directly. This method can handle large files better: 
```bash
sqlcmd -S servername -d databasename -i scriptfile.sql
```
